website/src/pages/downloads.tsx

```diff
@@ -103,18 +104,76 @@ const VersionSelect = () => {
     );
 };
 
-const Asset = ({ url }) => {
+const Asset = ({ urls }) => {
+    let asset: string;
+    let gpgSig: string;
+    let coCert: string;
+    let coSig: string;
+    let name: string = "Binary";
+
+    for (let url of urls) {
+        if (url.toLowerCase().includes(".deb")) {
+            name = "Debian Package";
+        } else if (url.toLowerCase().includes(".rpm")) {
+            name = "RPM Package";
+        } else if (url.toLowerCase().includes(".pkg") && url.toLowerCase().includes("linux")) {
+            name = "Arch Package";
+        }
+
+        if (url.toLowerCase().includes(".gpgsig")) {
+            gpgSig = url;
+            continue;
+        }
+        if (url.toLowerCase().includes(".pem")) {
+            coCert = url;
+            continue;
+        }
+        if (url.toLowerCase().includes(".sig")) {
+            coSig = url;
+            continue;
+        }
+        asset = url;
+    }
+
+    if (asset === undefined) {
+        console.trace("UNKOWN ASSET: ", asset, urls, gpgSig, coCert, coSig);
+        return;
+    }
+
     const { selectedItem } = useOptions();
     return (
-        <div className="pagination-nav__item">
-            <a className="pagination-nav__link" href={url}>
-                <div className="pagination-nav__label">
-                    {AssetArchitecture(url).toUpperCase()}
+        <div className="card download-card">
+            <div className="pagination-nav__item">
+                <div className="card__header">
+                    <h5>
+                        <a href={asset}>
+                            {AssetArchitecture(asset).toUpperCase()} - {name}
+                        </a>
+                    </h5>
                 </div>
-                <div className="pagination-nav__sublabel">
-                    Version: {selectedItem}
+                <div className="card__body">
+                    <p className="text--center">
+                        Version: {selectedItem}
+                    </p>
                 </div>
-            </a>
+                <div className="card__footer">
```

Comment:
Now the buttons are cut off in all cards for me:
![image](https://github.com/user-attachments/assets/ee65d245-98c7-4ecf-881a-41e773eb145e)

Reply:
@JanMa this should be fixed, let me know!

Reply:
Your latest changes seem to fix this :+1:

Reply:
Ok, I'll try again lol. I'm not sure how much we can do, given the buttons are from the framework and we probably just need bigger changes at some point...

Reply:
I've played around with it a bit and what works very well for smaller screen sizes is to just have one row of cards instead of two. Not sure yet what CSS magic one needs to use so that we'll still have two rows for bigger screens 😄

Reply:
@JanMa Can you stick it in a media query for like 720px screens and below?

Reply:
Yes, much better :-) One last small improvement suggestion:
![image](https://github.com/user-attachments/assets/6e27739b-e44a-4f88-9447-25378e0b3da2)
I think it looks much nicer if we force all corners of the buttons to be round

Reply:
Ahh, yes, I like.